02-peer-protocol.md

```diff
@@ -566,12 +579,17 @@ The funding node:
     - SHOULD send a `closing_signed` message.
 
 The sending node:
-  - MUST set `fee_satoshis` less than or equal to the
- base fee of the final commitment transaction, as calculated in [BOLT #3](03-transactions.md#fee-calculation).
-  - SHOULD set the initial `fee_satoshis` according to its
- estimate of cost of inclusion in a block.
-  - MUST set `signature` to the Bitcoin signature of the close
- transaction, as specified in [BOLT #3](03-transactions.md#closing-transaction).
+  - SHOULD set the initial `fee_satoshis` according to its estimate of cost of
+  inclusion in a block.
+  - SHOULD set `fee_range` according to the minimum and maximum fees it is
+  prepared to pay for a close transaction.
+  - if it doesn't receive a `closing_signed` response after a reasonable amount of time:
+    - MUST fail the channel
+  - if it is not the funder:
```

Comment:
Things seem undefined if the receiver finds the range sent over unacceptable. Say they want to close ASAP, and want a fee greater than the max fee proposed, what happens then? Do negotiations just break down and both sides need to reconnect and try again?

Reply:
You send a `warning` and then wait some time and then force-close. The "wait some time and then force-close" part is deliberately left to the implementation to decide how they want to do that, or, even, *if* they want to do that vs just making the user manually force-close.